src/justenoughharfbuzz.c

```diff
@@ -286,6 +286,61 @@ int shape (lua_State *L) {
     return glyph_count;
 }
 
+int get_glyph_dimensions(lua_State *L) {
+  size_t font_l;
+  const char * font_s = luaL_checklstring(L, 1, &font_l);
+  unsigned int font_index = (unsigned int)luaL_checknumber(L, 2);
+  double point_size = (unsigned int)luaL_checknumber(L, 3);
+  hb_codepoint_t glyphId = (hb_codepoint_t)luaL_checknumber(L, 4);
+
+  hb_blob_t* blob = hb_blob_create(font_s, font_l, HB_MEMORY_MODE_WRITABLE,
+      (void*)font_s, NULL);
+  hb_face_t* hbFace = hb_face_create(blob, font_index);
+  hb_font_t* hbFont = hb_font_create(hbFace);
+  unsigned int upem = hb_face_get_upem(hbFace);
+  hb_font_set_scale(hbFont, upem, upem);
+
+  /* Harfbuzz's support for OT fonts is great, but
+     there's currently no support for CFF fonts, so
```

Comment:
I'm not sure this is true any more (probably was when this code was originally written!). Instead use `can_use_ot_funcs`.